hospitals/utils.py

```diff
@@ -1,6 +1,6 @@
 from pymantic import sparql
 
-server = sparql.SPARQLServer('http://localhost:9999/blazegraph/namespace/kb/sparql')
+server = sparql.SPARQLServer('http://169.254.0.72:9999/blazegraph/namespace/kb/sparql')
```

Comment:
Maybe we can use environment variable instead?

Reply:
we should :v

Reply:
Change to localhost :v